lib/src/command/lish.dart

```diff
@@ -251,64 +271,120 @@ the \$PUB_HOSTED_URL environment variable.''',
       usageException('Cannot use both --force and --dry-run.');
     }
 
-    if (entrypoint.root.pubspec.isPrivate) {
+    if (_fromArchive != null && _toArchive != null) {
+      usageException('Cannot use both --from-archive and --to-archive.');
+    }
+
+    if (_fromArchive != null && dryRun) {
+      usageException('Cannot use both --from-archive and --dry-run.');
+    }
+
+    if (_toArchive != null && dryRun) {
+      usageException('Cannot use both --to-archive and --dry-run.');
+    }
+
+    if (_fromArchive != null && force) {
+      usageException('Cannot use both --from-archive and --force.');
```

Comment:
Why not? will `--from-archive` not prompt?

Reply:
Yeah - this is a mistake. Removed.